packages/@react-spectrum/s2/chromatic/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Breadcrumb, Breadcrumbs} from '../src';
+import {generatePowerset} from '@react-spectrum/story-utils';
+import {Many} from '../stories/Breadcrumbs.stories';
+import type {Meta, StoryObj} from '@storybook/react';
+import {shortName} from './utils';
+import {style} from '../style/spectrum-theme' with { type: 'macro' };
+import {userEvent, within} from '@storybook/testing-library';
+
+const meta: Meta<typeof Breadcrumbs> = {
+  component: Breadcrumbs,
+  tags: ['autodocs'],
+  title: 'S2 Chromatic/Breadcrumbs'
+};
+
+export default meta;
+
+export const Dynamic = Many as StoryObj;
+
+Dynamic.parameters = {
+  // TODO: move these options back to meta above once we get strings for ar-AE. This is just to prevent the RTL story's config from actually applying
```

Comment:
Do we have translations since we're about to release?

Reply:
i don't think we've gotten the translations for breadcrumbs yet